scripts/bundleSize/bundleSize.test.js

```diff
@@ -1,30 +1,57 @@
-import ora from 'ora';
-import chalk from 'chalk';
-import { readdirSync, statSync } from 'fs';
 import stripAnsi from 'strip-ansi';
+import { jest } from '@jest/globals';
+import pageTypeBundleExtractor from './__mocks__/pageTypeBundleExtractor.js';
 
-jest.mock('./pageTypeBundleExtractor');
-jest.mock('ora');
-jest.mock('chalk', () => ({
-  red: a => a,
-  green: a => a,
-  blue: a => a,
-  yellow: a => a,
-  cyan: { bold: b => b },
-  bold: a => a,
-}));
-jest.mock('../../cypress/support/config/services', () => ({
-  service1: {},
-  service2: {},
-}));
-jest.mock('fs');
-jest.mock('./bundleSizeConfig', () => ({
+jest.unstable_mockModule('./pageTypeBundleExtractor', () => (pageTypeBundleExtractor));
+
+jest.unstable_mockModule('./bundleSizeConfig', () => ({
   MIN_SIZE: 632,
   MAX_SIZE: 728,
 }));
 
+jest.unstable_mockModule(
+  './serviceList.js',
+  () => ({ default: ["service1", "service2"] }),
+);
+
+jest.unstable_mockModule('fs', () => ({
+  default: {
+    readdirSync: jest.fn(),
+    statSync: jest.fn(),
+  },
+}));
```

Comment:
`unstable_mockModule` calls require a factory function. They appear to always return a json object with a `default` key and any named exports as keys on the `default` json object.